### PR TITLE
updating vendor bump go-open-service-broker-client to master branch

### PR DESCRIPTION
**Pull request by shawn-hurley**
* needed to pull in new constants for platform field in
OriginatingIdentity.

### CODE REVIEW

**Comment by shawn-hurley** [comment]
FAIL	github.com/pmorie/osb-broker-lib/vendor/github.com/prometheus/procfs/sysfs	0.003s

It appears that we testing vendor. I don't know what the correct behavior is for this should be.

File: Gopkg.lock [hunk 1]
```diff
@@ -38,10 +38,10 @@
   version = "v1.0.0"
 
 [[projects]]
+  branch = "master"
```
Comment (pmorie) [inline Gopkg.lock]: We should pin only to released versions. I’ll do a release in the client.

**Comment by pmorie** [comment]
0.0.5 is released now

**Comment by LiliC** [comment]
Will let @pmorie have the final look and merge it, as he had some comments before.

**Comment by pmorie** [comment]
Superceded by #26

**Pull request closed without merging**